UCSD/campus-mobile
# Remove background location

## Summary
Remove background location


## Changelog

[General] [Remove] - Remove background location from iOS and Android


## Test Plan
- Ensure background location has been removed for iOS and Android
- Ensure foreground location works as expected for iOS and Android
